### DISCUSSION TITLE
Question: Can hybridSPAdes be used with Illumina RNA-seq along with both PacBio and Nanopore genomic reads?

### DISCUSSION THREAD

**Discussion by jolespin** [ablab/spades#750] [Q&A] [answered]
I am trying a de-novo assembly of a diatom and inherited a few different read sets.  I have several Illumina short read RNA-seq samples, a few PacBio genomic long reads, and a Nanopore genomic long read file.  

Is it possible to use all of these together with hybridSPAdes or does SPAdes expect short read genomic reads instead of RNA-seq to polish?

I also have some dropseq reads which have 57 bp reads at the 3' end of a gene...Not sure if I could use this to help with the assembly tho.

**Comment by andrewprzh** [comment]
Yes, hybridSPAdes expects only genomic data. We have an experimental rnaSPAdes pipeline which can use RNA-Seq Illumina + RNA PacBio / Nanopores. However, we have never tried combining RNA and genomic reads for the purpose of transcriptome assembly.

I'm not very familiar with diatom genome, but if it has rather simple gene structure (e.g. small number of introns), you may try to use existing hybrid rnaSPAdes pipeline. Genomic long reads that do not align to the genes will be just ignored in this case. Not sure how the pipeline will work in the presence of intons though.

You can obtain this experimental version by writing me directly to andrewprzh@gmail.com

Best
Andrey

**Comment by asl** [comment, marked as answer]
The latest SPAdes 3.15.2 does support hybrid RNA assemblies
